w3lib/url.py

```diff
@@ -264,3 +264,8 @@ def any_to_uri(uri_or_path):
         return path_to_file_uri(uri_or_path)
     u = urlparse(uri_or_path)
     return uri_or_path if u.scheme else path_to_file_uri(uri_or_path)
+
+
+__all__ = ["add_or_replace_parameter", "any_to_uri", "file_uri_to_path",
```

Comment:
what do you think about adding deprecated and semi-private function here as well?
`urljoin_rfc`, `is_url`, `url_query_parameter` are missing.